frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tier1-b/url-examples.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { TEMPLATE } from '../../utils/const';
+import { ProvisionSource } from '../../utils/const/provisionSource';
+import * as wizardView from '../../views/selector-wizard';
+import { wizard } from '../../views/wizard';
+
+describe('Import URL examples', () => {
+  before(() => {
+    cy.Login();
+    cy.visitVMsList();
```

Comment:
Let's creating a namespace here, otherwise non privileged user cannot visit the template list.

Reply:
ok